### ISSUE TITLE
Auto-naming for Item Variant

### ISSUE THREAD

**Issue by niethaheng** [frappe/erpnext#5496]
So when you try to make a variant, the Item Name will be automatically written as [item_code]-[item_variant] like this:
![screen shot 2016-06-17 at 11 13 40 am](https://cloud.githubusercontent.com/assets/19649986/16141766/97283f22-347c-11e6-8edd-bddc218f0155.png)

However, not all company use extensive item code. Example for our business, every item is very unique that the Generic Name is used for our documentation. I just made up new item code to comply with the system. However no one knows about the item code and don't see the importance.

So we are trying to maker the new item name as the [item_name]-[item_variant] automatically such as this:
![screen shot 2016-06-17 at 11 16 12 am](https://cloud.githubusercontent.com/assets/19649986/16141802/ec5cc044-347c-11e6-99fb-908e53ca41ea.png)

We can change it manually by typing. It is not a problem. Except if your portfolio includes this amount of item:
![screen shot 2016-06-17 at 11 17 09 am](https://cloud.githubusercontent.com/assets/19649986/16141811/1869b7d2-347d-11e6-992f-5b8b6be996d7.png)

I tried to convince the team to use export-import feature instead but it is proven to be very complicated for them. Note that for each item in Pharmaceutical we have up to 5 variants.

Is there any way I can make this more efficient?

Best regards,

Nietha

**Comment by cgpurbaugh** [comment]
Not really, this is the purpose of the import export tool.  I don't think you will find an easier alternative.
